docs/_jan-milestone_.md

```diff
@@ -0,0 +1,50 @@
+# Jan milestone
+
+## Overview
+
+- Take some time to:
+  - fix bugs, improve developer experience, fix docs
+  - triage open issues across all the repos, not just `loopback-next`
+- Migration work continues to be one of the focus
+
+## Goals
+
+- [ ] Improve developer experience
+
+  - [ ] [3] Expose isActive() method in Transaction interface #3471
+  - [ ] [3] Missing model.ts after running lb4 openapi command #4135
+  - [ ] [2] Model strict: false is ignored with Postgre SQL repository #4042
+  - [ ] [needs estimates] Column names in lowercase using lb4 discover method -
+        SQL connector #3343
+
+- [ ] Bugs
+
+  - [ ] [3] Docs: Improve TodoList Tutorial instructions and completed tutorial
+        'lb4 example todo-list' #4161
+
```

Comment:
I am proposing to add the following bug(s) to the Jan milestone:

- https://github.com/strongloop/loopback/issues/4287 lb3 - Relations on uuid

Reply:
- Preserve custom type of auto-generated id property #3602

Reply:
for #3602, will put it in the stretch goal. It seems like @frbuceta will pick it up.